Library/Homebrew/formula.rb

```diff
@@ -1308,8 +1308,6 @@ def to_hash
       }
     end
 
-    hsh["installed"] = hsh["installed"].sort_by { |i| Version.new(i["version"]) }
```

Comment:
This is obviously no longer needed because of the above change in `installed_kegs`, so I removed it.